src/main/java/com/github/iarellano/j2y2j/NoSpaceBeforeColonPrettyPrinter.java

```diff
@@ -0,0 +1,19 @@
+package com.github.iarellano.j2y2j;
+
+import com.fasterxml.jackson.core.util.DefaultPrettyPrinter;
+import com.fasterxml.jackson.core.util.Separators;
+
+public class NoSpaceBeforeColonPrettyPrinter extends DefaultPrettyPrinter {
+
+    @Override
+    public DefaultPrettyPrinter createInstance() {
+        return new NoSpaceBeforeColonPrettyPrinter();
+    }
+
+    @Override
+    public NoSpaceBeforeColonPrettyPrinter withSeparators(Separators separators) {
+        _separators = separators;
+        _objectFieldValueSeparatorWithSpaces = separators.getObjectFieldValueSeparator() + " ";
```

Comment:
The default pretty printer adds a space before and after the separator. I omitted the starting one.